modules/core/arrow-test/src/main/kotlin/arrow/test/generators/Generators.kt

```diff
@@ -107,6 +108,7 @@ fun Gen.Companion.intPredicate(): Gen<(Int) -> Boolean> =
     )
   }
 
+fun <A> Gen.Companion.endo(gen: Gen<A>): Gen<Endo<A>> = gen.map { a: A -> Endo<A> { a } }
```

Comment:
Kotlin test has an arrow module. We should add all these gen instances there if they make sense